specification/base/typespec/responses/operations.tsp

```diff
@@ -36,14 +36,19 @@ interface Responses {
   @route("{response_id}")
   getResponse(
     @path
-    @example("resp_677efb5139a88190b512bc3fef8e535d")
     response_id: string,
 
     @query(#{ name: "include[]", explode: true })
-    includables?: Includable[] = #[],
```

Comment:
WTF even is `#[]` ?

Reply:
It represents a default empty array in TypeSpec.